lib/blpapi-wrapper.ts

```diff
@@ -272,15 +286,19 @@ export class Session extends events.EventEmitter {
         }).nodeify(cb);
     }
 
-    request(uri: string, name: string, request: any, callback: IRequestCallback): void {
+    request(uri: string, requestName: string, request: any, callback: IRequestCallback): void {
         this.validateSession();
 
+        var responseEventName = REQUEST_TO_RESPONSE_MAP[requestName];
+        if (responseEventName === undefined) {
```

Comment:
Please change this test to be to protect against having an accidental mapping where the value is `undefined`:
```typescript
if (!(responseEventName in REQUEST_TO_RESPONSE_MAP)) {
    return callback(new Error('Request type ' + requestName + ' not handled'));
}
var responseEventName = REQUEST_TO_RESPONSE_MAP[requestName];
```